NEWS.md

```diff
@@ -19,6 +19,7 @@ in the augment method for the chi sq test, .residuals column was renamed to .res
 
 - augment
 - tidy.lsmobj gains a `conf.int` argument.
+- tidy.survreg gains a `conf.int` argument.
```

Comment:
Would you add a note here that previously the confidence interval was included by default but the new behavior does not include a confidence interval by default?

Reply:
Done - let me know if the phrasing is clear enough or needs reworking.